lib/Rex/Pkg/FreeBSD.pm

```diff
@@ -69,9 +75,21 @@ sub is_installed {
   Rex::Logger::debug(
     "Checking if $pkg" . ( $version ? "-$version" : "" ) . " is installed" );
 
+  # only need -g if pkg is a glob
+  my $glob_arg = '';
+  if ( $pkg =~ /\*/ ) {
+
+    # back quote asterisks so the command can actually be used
+    $pkg =~ s/\*/\\\*/g;
```

Comment:
Referencing the quoting method note from above.

Reply:
Done!